src/components/PostList.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import posts from '../api/posts';
+import comments from '../api/comments';
+import users from '../api/users';
+import Post from './Post';
+
+function PostList() {
+  return (
```

Comment:
```suggestion
function PostList() {
  const preparedPosts = posts.map((post) => ({
    post,
    user: users.find...
    comments: comments.filter...
  }));

  return (
```
Объедини все данные сразу в один массив

Reply:
Почему не объединил?